checkstyle.xml

```diff
@@ -0,0 +1,380 @@
+<?xml version="1.0"?>
```

Comment:
Can you please include any files/resources you used in reference when creating this? Also, can you reply to this comment with some context on how it was created? E.g. did you write it manually, did you use a tool to generate it, etc.

Reply:
I had taken the reference from the original file 
https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml

Reply:
removed checkstyle file.